tests/test_fmi3.py

```diff
@@ -548,6 +548,24 @@ def test_get_variable_data_type(self, variable_name, expected_datatype):
         fmu = FMUModelME3(fmu_path, _connect_dll = False)
         assert fmu.get_variable_data_type(variable_name) is expected_datatype
 
+    def test_simulate(self):
+        """Test basic simulation of an FMU, no result handling."""
```

Comment:
Also check the simulation statistics; they are identical, but I wouldn't want to regression test them here.